README.md

```diff
@@ -16,17 +16,24 @@ V3IO Frames (**"Frames"**) is a multi-model open-source data-access library, dev
 
 - [Overview](#overview)
 - [User Authentication](#user-authentication)
-- [Client Constructor](#client-constructor)
-- [Common Client Method Parameters](#client-common-method-params)
-- [create Method](#method-create)
-- [write Method](#method-write)
-- [read Method](#method-read)
-- [delete Method](#method-delete)
-- [execute Method](#method-execute)
+- [`Client` Constructor](#client-constructor)
+- [Common `Client` Method Parameters](#client-common-method-params)
+- [`create` Method](#method-create)
+- [`write` Method](#method-write)
+- [`read` Method](#method-read)
+- [`delete` Method](#method-delete)
+- [`execute` Method](#method-execute)
 
 <a id="api-reference-overview"></a>
 ### Overview
 
+- [Initialization](#initialization)
+- [`Client` Methods](#client-methods)
+- [Backend Types](#backend-types)
```

Comment:
I think `Backend Types` should be before `Client Methods` since the client methods refers to specific backend types that the user "dont know of yet"

Reply:
ACCEPT — I moved the **"Backend Types"** overview section before the **"Client Methods"** section and I edited the backend-specific method params note and moved it to the "Client Methods" section. See commit b4c4460.